patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -1,5 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/EntityRenderer.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/EntityRenderer.java
+@@ -199,7 +199,7 @@
+             else if (p_175066_1_ instanceof EntityEnderman)
+             {
+                 this.func_175069_a(new ResourceLocation("shaders/post/invert.json"));
+-            }
++            } else {ResourceLocation r = net.minecraftforge.fml.client.registry.ClientRegistry.getEntityShader(p_175066_1_.getClass()); if (r != null) this.func_175069_a(r);}
```

Comment:
Committing patches that are JUST line number changes is bad.
Or when things are 1 line change and 50 line number changes.
But this one is fine.